tests/App/Http/Controllers/Filament/LogoutControllerTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Models\User;
+use Illuminate\Support\Facades\DB;
+
+it('should redirect the admin user to homepage on logout', function () {
+    $user = User::factory()->create();
+
+    DB::insert(
+        'insert into model_has_roles (role_id, model_type, model_id) values (?, ?, ?)',
+        [1, 'App\\Models\\User', $user->id]
```

Comment:
```suggestion
        [1, User::class, $user->id]
```

Reply:
also think you can also do something like `$user->assignRole(Role::Superadmin)` to avoid these raw insert queries